src/toil/resource.py

```diff
@@ -287,11 +287,21 @@ def _load(cls, path):
         else:
             # This is a simple user script (with possibly a few helper files)
             rootDir = path
-        with ZipFile(file=bytesIO, mode='w') as zipFile:
-            for dirName, _, fileList in os.walk(path):
-                for fileName in fileList:
-                    fullPath = os.path.join(dirName, fileName)
-                    zipFile.write(fullPath, os.path.relpath(fullPath, rootDir))
+        blacklistDir = ['/tmp', '/var', '/etc', '/bin', '/sbin', '/home', '/dev', '/sys', '/usr', '/run']
```

Comment:
let's call this the skipdirList, that'd be clearer

Reply:
got it! updated it in new commit